src/pages/components/feature-card-block.mdx

```diff
@@ -22,16 +22,39 @@ import ComponentFeedback from 'components/ComponentFeedback';
 ## Feature card block &mdash; medium
 Feature card block &mdash; medium is used to present a group of information with a medium size card with an image.
 
+### Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes   | 1 | XS (65 / 85)|
+| [Featured card](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/feature-card) | Component | Yes  | 1 | - |
+
+To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank">link</a>.
+
+
 ![Feature card block medium](../../images/pattern/feature-card-block/feature-card-block-medium.jpg)
 
 ## Feature card block &mdash; large
 Feature card block &mdash; large is used to present main piece of content as a main feature.
 
+### Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Eyebrow | Short copy | Yes   | 1 | Mini (25 / 35)|
+| Headline | Short copy | Yes   | 1 | XS (65 / 85)|
+| Paragraph | Long copy | No   | 1 | LL (200 / 260)|
+| Image | Media | Yes   | 1 | - |
+| URL | Link | Yes   | 1 | - |
+| [CTA type](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) (ex. Local) | Option list | Yes  | 1 | - |
```

Comment:
I would call it just type personally. This was an odd one that looking back at the documentation it might still need some work. It mirrors the CTA types, but it's not included in with the other CTAs. One way to look at this is maybe it's CTA style feature then it'd adopt all the types, but I'm not convinced development built it out like that.

Reply:
I like "Type". I'll switch it to that.